packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -121,6 +121,8 @@ CheckoutStepHeader.propTypes = {
 };
 
 function Stepper( { isComplete, isActive, className, children } ) {
+	// Prevent showing complete stepper when active
+	isComplete = isActive ? false : isComplete;
 	return (
 		<StepNumberOuterWrapper className={ joinClasses( [ className, 'checkout-step__stepper' ] ) }>
 			<StepNumberInnerWrapper isComplete={ isComplete }>
```

Comment:
Does it make sense to call `isComplete` something like `isCompleteAndActive` here? I can imagine myself being confused that a prop with the same name has a different value.

Reply:
That's a valid point. It would actually be `isCompleteAndNotActive`, which shows already how this can be confusing in its current form.